### ISSUE TITLE
Error thrown on every export

### ISSUE THREAD

**Issue by godsyn** [Cacti/plugin_gexport#51]
Error thrown on fresh install of cacti 1.2.17, gexport version 2.0

```
<!--StartFragment-->
2021-06-10 13:20:08 - EXPORT MASTER STATS: Time:6.64 Exports:1 Exported:1
--
2021-06-10 13:20:08 - EXPORT STATS: ExportID:1  (/var/www/cactiexportsync) ExportDate:2021-06-10_13:20:08  ExportDuration:6.64 TotalGraphsExported:40 MaximumThreads: 15
2021-06-10 13:20:08 - CMDPHP PHP ERROR NOTICE Backtrace:   (
	/plugins/gexport/poller_export.php[107]:graph_export(),  
	/plugins/gexport/functions.php[120]:run_export(),
	/plugins/gexport/functions.php[162]:exporter(),
	/plugins/gexport/functions.php[422]:tree_site_export(),
	/plugins/gexport/functions.php[1921]:export_generate_site_html(),
	/plugins/gexport/functions.php[1824]:get_formatted_data_query_indexes(),
	/lib/data_query.php[1793]:CactiErrorHandler()
)
2021-06-10 13:20:08 - ERROR PHP NOTICE: Undefined index: sort_field  in file: /var/www/cacti/lib/data_query.php  on line: 1793
2021-06-10 13:20:01 - SYSTEM STATS: Time:1.2129 Method:spine  Processes:1 Threads:1 Hosts:12 HostsPerProcess:12 DataSources:23  RRDsProcessed:7

<!--EndFragment-->
```

**Comment by netniV** [comment]
This suggests that there was a bad link or damaged graph.  Was it exporting a damaged graph?

**Comment by godsyn** [comment]
Can you walk me through the process for obtaining the debug info you
require, please? There are currently only a handful of graphs being
exported.

The above error is all that is supplied in the logs, and occurs every
export.

**Comment by netniV** [comment]
So it turns out that this particular problem is not with your data but an issue in the data_query.php file.  This has already been fixed for 1.2.18 that is upcoming in commit cacti/cacti@c5c088d2d
